sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/handler/ReceiverUnsettledDeliveries.java

```diff
@@ -697,7 +697,7 @@ Throwable getRejectedOutcomeError() {
          *
          * @return {@code true} if the work has timed out, {@code false} otherwise.
          */
-        boolean hasTimedout() {
+        boolean hasTimedOut() {
```

Comment:
Package private method, per lint suggestion renaming to be consistent with WindowWork::hasTimedOut naming.